lib/reporters/base.js

```diff
@@ -368,105 +354,124 @@ function pad (str, len) {
 }
 
 /**
- * Returns an inline diff between 2 strings with coloured ANSI output
+ * Converts an inline diff between two strings to coloured text representation.
  *
  * @api private
- * @param {Error} err with actual/expected
- * @return {string} Diff
+ * @param {Object} diff structured differences actual from expected
+ * @return {string} The diff for output to the console.
  */
-function inlineDiff (err) {
-  var msg = errorDiff(err);
+function inlineDiff (diff) {
+  var indent = '      ';
+  var lines = [''];
+  diff.forEach(function (change) {
+    function colorize (line) {
+      if (change.added) {
+        return color('diff added', line);
+      }
+      if (change.removed) {
+        return color('diff removed', line);
+      }
+      return line;
+    }
+
+    change.value.split('\n').forEach(function (line, i) {
+      line = colorize(line);
+      if (i === 0) {
```

Comment:
what's this?

Reply:
This prevent adding delimiters between hunks before first hunk